src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -277,7 +277,7 @@ rec_free(struct d_hash_table *htable, d_list_t *rlink)
 	if (rc == ENOMEM)
 		rc = dfs_release(hdl->oh);
 	if (rc)
-		D_ERROR("dfs_release() failed: %d (%s)", rc, strerror(rc));
+		D_ERROR("dfs_release() failed: %s\n", strerror(rc));
```

Comment:
The standard we've settles on for D_ERROR is as below, there are places we test this but adding coverage is slow and obviously pil4dfs is not yet covered.
```suggestion
		D_ERROR("dfs_release() failed: %d (%s)\n", rc, strerror(rc));
```
The only think wrong with this line before your change was the lack of \n, all D_ERROR and D_WARN calls should have precisely one \n at the end.

Reply:
Fixed as you suggested. Thank you!